pvlib/iotools/pvgis.py

```diff
@@ -383,11 +383,17 @@ def read_pvgis_hourly(filename, pvgis_format=None, map_variables=True):
                     fbuf, map_variables=map_variables)
         return pvgis_data
 
-    # raise exception if pvgis format isn't in ['csv', 'json']
-    err_msg = (
-        "pvgis format '{:s}' was unknown, must be either 'json' or 'csv'")\
-        .format(outputformat)
-    raise ValueError(err_msg)
+    elif outputformat == 'basic':
```

Comment:
I think this condition and these changes should be moved to `get_pvgis_tmy()` instead of here in `read_pvgis_hourly()` which does not accept `basic` as input to the `pvgis_format` argument

Reply:
Good catch!